examples/chart/teleport-cluster/templates/clusterrole.yaml

```
@@ -24,7 +25,7 @@ rules:
   verbs:
   - create
 
-{{ if .Values.operator.enabled }}
+{{- if .Values.operator.enabled -}}
```

Tests are failing because of this line, trailing on both sides of the template instruction causes the template to render like:
```
  1244	---$
  1245	# Source: teleport-cluster/templates/clusterrole.yaml$
  1246	apiVersion: rbac.authorization.k8s.io/v1$
  1247	kind: ClusterRole$
  1248	metadata:$
  1249	  name: test$
  1250	rules:$
  1251	- apiGroups:$
   # [...]
  1267	  resources:$
  1268	  - selfsubjectaccessreviews$
  1269	  verbs:$
  1270	  - create- apiGroups:$
  1271	  - "resources.teleport.dev"$
  1272	  resources:$
  1273	  - teleportroles$
  1274	  - teleportroles/status$
  1275	  - teleportusers$
  1276	  - teleportusers/status$
  1277	  verbs:$
  1278	  - get$
  1279	  - list$
  1280	  - patch$
  1281	  - update$
  1282	  - watch$
  # [...]
  ```
  
  ```suggestion
{{- if .Values.operator.enabled }}
```